operations/iuf/workflows/configuration.md

```diff
@@ -51,6 +51,7 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
 
 1. Edit the `compute-and-uan-bootprep.yaml` and `management-bootprep.yaml` files to account for any site deviations from the default values. For example:
     - Comment out the `slurm-site` CFS configuration layer and uncomment the `pbs-site` CFS configuration layer in `compute-and-uan-bootprep.yaml` if PBS is the preferred workload manager
+    - Comment out the SBPS `rootfs_provider` and `rootfs_provider_passthrough` parameters and uncomment the CPS `rootfs_provider` and `rootfs_provider_passthrough` parameters, if CPS is the preferred method to project content.
```

Comment:
if DVS with CPS is the preferred method to project content?

Reply:
Changed.